<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ddiview</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QSplitter" name="splitDdiDdb">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="grpDdi">
       <property name="title">
        <string>DDI</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QTreeWidget" name="treeStructure">
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <attribute name="headerCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Chunk</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="lblProperties">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Item Properties</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnShowDdb">
                <property name="text">
                 <string>Show DDB</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listProperties">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="textElideMode">
               <enum>Qt::ElideNone</enum>
              </property>
              <property name="flow">
               <enum>QListView::TopToBottom</enum>
              </property>
              <property name="isWrapping" stdset="0">
               <bool>false</bool>
              </property>
              <property name="resizeMode">
               <enum>QListView::Fixed</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="grpDdb">
       <property name="title">
        <string>DDB</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSplitter" name="splitDdb">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QTreeWidget" name="treeStructureDdb">
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <attribute name="headerCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Chunk</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="lblDdbProperties">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Item Properties</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnShowMediaTool">
                <property name="text">
                 <string>Show Media Tool</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listDdbProperties">
              <property name="textElideMode">
               <enum>Qt::ElideNone</enum>
              </property>
              <property name="flow">
               <enum>QListView::TopToBottom</enum>
              </property>
              <property name="isWrapping" stdset="0">
               <bool>false</bool>
              </property>
              <property name="resizeMode">
               <enum>QListView::Fixed</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="grpMediaTool">
              <property name="title">
               <string>Media Tool</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QStackedWidget" name="stackedWidget">
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="synthesis"/>
                 <widget class="QWidget" name="playback"/>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExportJson"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>Statistics</string>
    </property>
    <addaction name="actionPropDist"/>
    <addaction name="actionArticulationTable"/>
    <addaction name="actionactionExportDdbLayout"/>
   </widget>
   <widget class="QMenu" name="menuExtraction">
    <property name="title">
     <string>Extraction</string>
    </property>
    <addaction name="actionExtractAllSamples"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuStatistics"/>
   <addaction name="menuExtraction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPropDist">
   <property name="text">
    <string>Property Distribution</string>
   </property>
   <property name="toolTip">
    <string>Get the distribution of a property's values over children of certain chunks.</string>
   </property>
  </action>
  <action name="actionArticulationTable">
   <property name="text">
    <string>Articulation Table</string>
   </property>
   <property name="toolTip">
    <string>Show the possible articulation combinations included in the database</string>
   </property>
  </action>
  <action name="actionExportJson">
   <property name="text">
    <string>Export to JSON</string>
   </property>
  </action>
  <action name="actionExtractAllSamples">
   <property name="text">
    <string>Extract all samples</string>
   </property>
  </action>
  <action name="actionactionExportDdbLayout">
   <property name="text">
    <string>DDB Layout</string>
   </property>
   <property name="toolTip">
    <string>Export the chunk layout in DDB (per shown by DDI).</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
